analyzer/src/main/kotlin/managers/PIP.kt

```diff
@@ -385,7 +398,22 @@ class PIP(analyzerConfig: AnalyzerConfiguration, repoConfig: RepositoryConfigura
         // Create an out-of-tree virtualenv.
         log.info { "Creating a virtualenv for the '${workingDir.name}' project directory..." }
         val virtualEnvDir = createTempDir(workingDir.name.padEnd(3, '_'), "virtualenv")
-        ProcessCapture(workingDir, "virtualenv", virtualEnvDir.path).requireSuccess()
+
+        //Create virtual environment with correct pip version. Basically we don't know where Pythons were installed.
+        //By default on Linux commands "python" and "python2" are using for Python version 2. After install Python version >3, should be active also command "python3".
+        //On Windows we have to add Python interpreters to environment variables and they should have another names.
+        //See: https://stackoverflow.com/a/22626734/5877109
+        val projectHasPython3 = ProjectVersion.isPython3(workingDir)
```

Comment:
How about renaming `projectHasPython3` to `isPython3Compatible`?

Reply:
I've improved the python_compatibility script a little and now returns the Python version instead of true / false - is it version 3? Currently this variable is called "projectPythonVer". What do you think about it?

Reply:
I think that's fine.